src/org/zaproxy/zap/extension/savexmlmessage/ZapAddOn.xml

```diff
@@ -0,0 +1,21 @@
+<zapaddon>
+	<name>Save XML Message</name>
+	<version>1</version>
+	<status>release</status>
```

Comment:
This should start as alpha.

Reply:
```<version>0.1</version>``` and ```<status>alpha</status>``` are ok?

Reply:
Bump, version needs to be an integer or semantic version.